src/shapes/Object/Object.ts

```diff
@@ -1703,10 +1706,10 @@ export class FabricObject<
    * @param {Boolean} [options.withoutShadow] Remove current object shadow. Introduced in 2.4.2
    * @return {Image} Object cloned as image.
    */
-  cloneAsImage(options: any) {
-    const canvasEl = this.toCanvasElement(options);
-    // TODO: how to import Image w/o an import cycle?
-    return new fabric.Image(canvasEl);
+  cloneAsImage(options: any): Image | null {
```

Comment:
I can't accept this.
Let's remove it entirely.
So much ugliness for a method of 2 lines

Reply:
for now it stays like this, we can reiterate.
I don't think cloneAsImage is that useful.
I also don't think many utils are that useful, but removing them now or stop exposing them now creates issues with tests
I m putting back things right now more than removing them, i was hoping i could just move around imports rather than change all of this